sway-lib-std/src/alias.sw

```diff
@@ -0,0 +1,6 @@
+library;
+
+/// The `AssetId` type is simply an alias for `b256` that represents the ID of a native asset.
+pub type AssetId = b256;
```

Comment:
I'm fine with this for now, but we should create a real newtype for AssetId similar to how ContractId is setup in a future release (which is basically also just a b256). This will likely break the SDK though, so it won't be a trivial change.

Reply:
A defined `AssetId` type would be beneficial. For now, I've added this to distinguish between when AssetId and SubId are needed in token operations.